File: tests/consumer-fixture/full/java/Vuln.java
import java.sql.*;
public class Vuln {
    public void query(Connection c, String userInput) throws SQLException {
        // semgrep p/java: SQL injection
        Statement s = c.createStatement();
        s.executeQuery("SELECT * FROM t WHERE n='" + userInput + "'");
    }
    public void exec(String cmd) throws Exception {
        Runtime.getRuntime().exec(cmd);   // command injection
    }
}

File: tests/consumer-fixture/full/dotnet/Vuln.cs
using System.Data.SqlClient;
public class Vuln {
    public void Query(string userInput) {
        // semgrep p/csharp: SQL injection via concatenation
        var sql = "SELECT * FROM Users WHERE Name = '" + userInput + "'";
        using var conn = new SqlConnection("Server=x;Database=y;User Id=sa;Password=P@ssw0rd;");
        using var cmd = new SqlCommand(sql, conn);
        cmd.ExecuteNonQuery();
    }
}
